TemporaryEmail/source/info.plist

```diff
@@ -110,6 +112,7 @@ readonly front_browser="$(osascript -e 'tell application "System Events" to retu
 
 if [[ "${front_browser}" == 'Safari'* || "${front_browser}" == 'Webkit'* ]]; then
   osascript &lt;&lt;EOF
+    tell application "System Events" to keystroke "${email}"
```

Comment:
Useful for me since I usually open the workflow when I'm in an input where I don't want to put my real email.

Reply:
I can see the usefulness, but I don’t agree with the implementation. Typing via AppleScript is finicky. It should instead output the email to a [Copy to Clipboard Output](https://www.alfredapp.com/help/workflows/outputs/copy-to-clipboard/) which can also paste it to the frontmost app. I’m making the change locally now.

Reply:
I disagree. Typing via AppleScript works great as long as you're not using special characters such as emojis. The generated email addresses never contain anything that wouldn't work. Copying to clipboard is a pain because Alfred doesn't give provide any good way to restore the clipboard. So you have to do it manually by storing the previous clipboard in a env variable, and then restoring it via e.g. another Alfred action, Apple script or some shell commands. I'm concerned it won't play well with some clipboard managers, and it seems like a more fragile solution prone to race conditions (if the clipboard is modified between the time you save it and the time your restore it).

Reply:
> works great as long as

I’ve seen AppleScript typing screw up in uppercases. I wouldn’t be surprised if it screwed up on `@`.

> Alfred doesn't give provide any good way to restore the clipboard.

Tell it to copy `{clipboard:1}`. Done.

Regardless of your feelings about copying to the clipboard, that’s what the Workflow was doing already and you didn’t remove that part. All I’m doing is moving the copy logic to another object so it can be pasted at the same time. Plus, I want the email in clipboard. Often I have to reference it again a short time after, so having it in the history is a feature.

Reply:
@vitorgalvao Need to use `{clipboard:0}` instead if the item is marked as transient.

In addition it doesn't work with non-textual clipboard content e.g. images.